src/pages/user/register/service.ts

```diff
@@ -0,0 +1,9 @@
+import request from '@/utils/request';
+import { UserRegisterParams } from './index';
+
+export async function fakeRegister(params: UserRegisterParams) {
+  return request('/api/register', {
+    method: 'POST',
+    data: params,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'data' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)